ads/appnexus.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  loadScript, writeScript, validateDataExists,
+}
+from '../src/3p';
+
+const APPNEXUS_AST_URL = 'https://acdn.adnxs.com/ast/ast.js';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function appnexus(global, data) {
+  const args = [];
+  args.push('size=' + data.width + 'x' + data.height);
+  if (data.tagid) {
+    validateDataExists(data, ['tagid']);
+    args.push('id=' + encodeURIComponent(data.tagid));
+    writeScript(global, constructTtj(args));
+    return;
+  } else if (data.member && data.code) {
+    validateDataExists(data, ['member', 'code']);
+    args.push('member=' + encodeURIComponent(data.member));
+    args.push('inv_code=' + encodeURIComponent(data.code));
+    writeScript(global, constructTtj(args));
+    return;
+  }
+
+  function constructTtj(args) {
+    let url = 'https://ib.adnxs.com/ttj?';
+    for (i = 0; i < args.length; i++) {
+      url += args[i] + '&';
```

Comment:
@cramforce 
They are already encoded above (line 33,38,39). I don't want to encode the `=`.